packages/db/db-client-types.ts

```diff
@@ -145,7 +148,7 @@ export type ContentItem = {
   created?: definitions['upload']['inserted_at']
   pins: Array<{
     status: definitions['pin']['status']
-    updated: definitions['pin']['updated_at']
+    // updated: definitions['pin']['updated_at']
```

Comment:
@GaryHomewood quick question.
Would you mind expanding around what's the rationale here?
If it's something that is not needed shall we go and remove it altogether?

Reply:
Ok just removed from the `ContentInput` type since it shouldn't need to be specified on create.